### PR TITLE
port to twisted.logger

### PR DESCRIPTION
**Pull request by NotAFile**

### CODE REVIEW

**Comment by NotAFile** [comment]
Note that this will mean any legacy scripts using "print" won't log anymore.

File: piqueserver/core_commands/server.py [hunk 1]
```diff
@@ -1,4 +1,15 @@
+import random
+
+from twisted.logger import Logger
+
 from piqueserver.commands import command, join_arguments
+from piqueserver.config import config
+
+log = Logger()
+
+name_option = config.option(
```
Comment (swalladge) [inline piqueserver/core_commands/server.py]: note: this is declared but not used in this file

Comment (NotAFile) [reply piqueserver/core_commands/server.py]: Oh, this was a remnant from before I added set_server_name

**Comment by NotAFile** [comment]
should be good now

**Comment by godwhoa** [comment]
Neat. Can we make the log level configurable?

**Comment by NotAFile** [comment]
@godwhoa yes, that would be a good next step

**Pull request merged**